### PR TITLE
Feature/more tests to improve coverage

### LINKED ISSUE #12: Implementar casos de teste relevantes que melhorem a cobertura de código do projeto original

### PR DESCRIPTION
**Pull request by lsr94**
## What's changed?
- Adding more tests to improve coverage
- New tests in the modules: Alert, Common e Warehouse

### CODE REVIEW

File: alerter/src/test/java/org/dromara/hertzbeat/alert/controller/AlertConvergeControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class AlertConvergeControllerTest {
```
Comment (isarbela) [inline alerter/src/test/java/org/dromara/hertzbeat/alert/controller/AlertConvergeControllerTest.java]: Remover o public (Issue Sonar)

**Review by isarbela — CHANGES_REQUESTED** [review]
https://sonarcloud.io/project/issues?resolved=false&pullRequest=33&id=karolalencar_ES2_2023-2_Hertzbeat&open=AY2OJzx78YUq_STiPRNU

**Review by isarbela — APPROVED** [review]
Ok

**Review by karolalencar — APPROVED** [review]
Revisado

**Pull request merged**